cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/BrokerCapacityConfigFileResolver.java

```diff
@@ -156,17 +156,18 @@
 
   @Override
   public void configure(Map<String, ?> configs) {
-    _configFile = KafkaCruiseControlUtils.getRequiredConfig(configs, CAPACITY_CONFIG_FILE);
+    String configFile = KafkaCruiseControlUtils.getRequiredConfig(configs, CAPACITY_CONFIG_FILE);
+    _configFile = configFile;
     try {
-      loadCapacities(_configFile);
+      loadCapacities(configFile);
```

Comment:
Sorry if my previous comment was not clear -- let me rephrase my question:

* Why are we passing a parameter to `loadCapacities` function? This is a local private function and we already have the value assigned to `_configFile`. Is it possible to modify `loadCapacities` to have no parameter and use `_configFile` in its implementation?